### ISSUE TITLE
1.46 error Global symbol "$icurve" requires explicit package name at /usr/local/Cellar/feedgnuplot/1.46/bin/feedgnuplot line 931.

### ISSUE THREAD

**Issue by ilovezfs** [dkogan/feedgnuplot#33]
1.45 is OK
```
iMac-TMP:~ joe$ echo -n "seq 5" | /usr/local/Cellar/feedgnuplot/1.45/bin/feedgnuplot --terminal 'dumb 80,20' --exit

                                                                                
    5 +---------------------------------------------------------------------+   
      |             +             +             +             +             |   
  4.5 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
    4 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
  3.5 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
    3 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
  2.5 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
    2 |-+...........:.............:.............:.............:...........+-|   
      |             :             :             :             :             |   
  1.5 |-+...........:.............:.............:.............:...........+-|   
      |             +             +             +             +             |   
    1 +---------------------------------------------------------------------+   
      0            0.2           0.4           0.6           0.8            1   
```

1.46 is not.
```                                                                                
iMac-TMP:~ joe$ echo -n "seq 5" | /usr/local/Cellar/feedgnuplot/1.46/bin/feedgnuplot --terminal 'dumb 80,20' --exit
Global symbol "$icurve" requires explicit package name at /usr/local/Cellar/feedgnuplot/1.46/bin/feedgnuplot line 931.
Execution of /usr/local/Cellar/feedgnuplot/1.46/bin/feedgnuplot aborted due to compilation errors.
iMac-TMP:~ joe$ 
```

**Comment by dkogan** [comment]
Thanks for the report! I'm a little puzzled at how I messed this up. How'd you find this so quickly? Do you have some script or something? In any case, fixed in 1.47.

**Comment by ilovezfs** [comment]
You're welcome!

>How'd you find this so quickly? Do you have some script or something?

Yes, we run https://github.com/Homebrew/homebrew-livecheck every few hours.
